### ISSUE TITLE
💡[FEATURE]: Add Cryptocurrency Price Prediction with Machine Learning

### ISSUE THREAD

**Issue by pavitraag** [Akshat111111/Hedging-of-Financial-Derivatives#691]
Predicting the price of cryptocurrencies is one of the popular case studies in the data science community. The prices of stocks and cryptocurrencies don’t just depend on the number of people who buy or sell them. Today, the change in the prices of these investments also depends on the changes in the financial policies of the government regarding any cryptocurrency. The feelings of people towards a particular cryptocurrency or personality who directly or indirectly endorse a cryptocurrency also result in a huge buying and selling of a particular cryptocurrency, resulting in a change in prices.

In short, buying and selling result in a change in the price of any cryptocurrency, but buying and selling trends depend on many factors. Using machine learning for cryptocurrency price prediction can only work in situations where prices change due to historical prices that people see before buying and selling their cryptocurrency. So, in the section below, I will take you through how you can predict the bitcoin prices (which is one of the most popular cryptocurrencies) for the next 30 days.
